pkg/gits/github.go

```diff
@@ -445,7 +445,13 @@ func (p *GitHubProvider) CreateWebHook(data *GitWebHookArguments) error {
 	}
 
 	log.Logger().Infof("Creating GitHub webhook for %s/%s for url %s", util.ColorInfo(owner), util.ColorInfo(repo), util.ColorInfo(webhookUrl))
-	_, _, err = p.Client.Repositories.CreateHook(p.Context, owner, repo, hook)
+	_, resp, err := p.Client.Repositories.CreateHook(p.Context, owner, repo, hook)
+	if err != nil {
+		if resp.StatusCode == 404 || resp.StatusCode == 403 {
+			log.Logger().Error("Permission denied")
```

Comment:
Why don't you wrap the error with a message in this case? 

```Go
errors.Wrap(err, "Permission denied")
```

an the let the caller the log the error

Reply:
That's more consistent with the rest of the code base.

Reply:
Maybe we can find a way to add a test as well? ;-)

Reply:
Yes, seems better.

Reply:
I'm keeping the statusCode just to make sure it appears only in case of 404 and 403.